agent/util-scripts/gold/pbench-copy-results/test-31.txt

```diff
@@ -0,0 +1,354 @@
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/opt/pbench-agent/unittest-scripts/ssh -q -i /var/tmp/pbench-test-utils/opt/pbench-agent/id_rsa -o StrictHostKeyChecking=no pbench@server.com exit
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/opt/pbench-agent/unittest-scripts/ssh -i /var/tmp/pbench-test-utils/opt/pbench-agent/id_rsa -o StrictHostKeyChecking=no pbench@server.com mkdir -p /foo/bar/testhost
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/opt/pbench-agent/unittest-scripts/ssh -i /var/tmp/pbench-test-utils/opt/pbench-agent/id_rsa -o StrictHostKeyChecking=no pbench@server.com mkdir -p /foo/bar/testhost
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/opt/pbench-agent/unittest-scripts/ssh -i /var/tmp/pbench-test-utils/opt/pbench-agent/id_rsa -o StrictHostKeyChecking=no pbench@server.com test -f /foo/bar/testhost/pbench-user-benchmark_ndk-test-1_2018.05.23T03.21.32.tar.xz
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/opt/pbench-agent/unittest-scripts/scp -o StrictHostKeyChecking=no -i /var/tmp/pbench-test-utils/opt/pbench-agent/id_rsa -o StrictHostKeyChecking=no ./pbench-user-benchmark_ndk-test-1_2018.05.23T03.21.32.tar.xz ./pbench-user-benchmark_ndk-test-1_2018.05.23T03.21.32.tar.xz.md5 /var/tmp/pbench-test-utils/pbench/tmp/prefix.pbench-user-benchmark_ndk-test-1_2018.05.23T03.21.32 pbench@server.com:/foo/bar/testhost
+/var/tmp/pbench-test-utils/test-execution.log:pbench@server.com:/foo/bar/testhost/iostat:foo
+/var/tmp/pbench-test-utils/test-execution.log:pbench@server.com:/foo/bar/testhost/mpstat:bar
+/var/tmp/pbench-test-utils/test-execution.log:pbench@server.com:/foo/bar/testhost/perf:baz
+/var/tmp/pbench-test-utils/test-execution.log:pbench@server.com:/foo/bar/testhost/pidstat:hunoz
+/var/tmp/pbench-test-utils/test-execution.log:pbench@server.com:/foo/bar/testhost/proc-interrupts:hukerz
+/var/tmp/pbench-test-utils/test-execution.log:pbench@server.com:/foo/bar/testhost/proc-vmstat:foobar
+/var/tmp/pbench-test-utils/test-execution.log:pbench@server.com:/foo/bar/testhost/sar:foo
+/var/tmp/pbench-test-utils/test-execution.log:pbench@server.com:/foo/bar/testhost/turbostat:bar
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/opt/pbench-agent/unittest-scripts/ssh -i /var/tmp/pbench-test-utils/opt/pbench-agent/id_rsa -o StrictHostKeyChecking=no pbench@server.com cd /foo/bar/testhost; md5sum --check pbench-user-benchmark_ndk-test-1_2018.05.23T03.21.32.tar.xz.md5
+/var/tmp/pbench-test-utils/test-execution.log:/var/tmp/pbench-test-utils/opt/pbench-agent/unittest-scripts/ssh -i /var/tmp/pbench-test-utils/opt/pbench-agent/id_rsa -o StrictHostKeyChecking=no pbench@server.com /opt/pbench-server/bin/pbench-server-set-result-state /foo/bar/testhost pbench-user-benchmark_ndk-test-1_2018.05.23T03.21.32.tar.xz
+--- test-execution.log file contents
+[pbench]
```

Comment:
Can you  add a "`+++ metadata.log file contents`" prefix and a "`--- metadata.log file contents`" suffix around the metadata.log contents?

Thanks!